workitem/expression_compiler.go

```diff
@@ -151,5 +151,5 @@ func (c *expressionCompiler) convertToString(value interface{}) (string, error)
 	default:
 		return "", fmt.Errorf("unknown value type of %v: %T", value, value)
 	}
-	return result, nil
+	return "[" + result + "]", nil
```

Comment:
@aslakknutsen No, sorry my bad, i didn't check that, the test is failing because of that, fixing it.